rust/worker/src/compactor/task.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(Clone, Eq, PartialEq)]
```

Comment:
a pattern we've followed for mod-level types is a types.rs file, maybe useful to copy here